### DISCUSSION TITLE
Allow locale parameter for Facebook provider

### DISCUSSION THREAD

**Discussion by skyfox69** [mpratt/Embera#93] [Ideas]
Hello,
I stumbled about randomly wrong languages for Facebooks like-, comment- and share-buttons. As using your plugin for a German web-site, the info-texts were displayed in English (best case), Arabic, Japanese or other.

Searching in Facebooks graph-API I detected the 'locale' parameter. Tweaking your code locally on my machine and working as a charm I thought you might think about adding 'locale' to the allowed parameters for the Facebook provider.

**Comment by mpratt** [comment]
Hi @skyfox69 
Thanks for the report! I'll be looking into this and pushing a new update probably tomorrow!

**Reply by skyfox69** [reply]
Hello @mpratt 
That would be nice. Please let me know about your decision

**Comment by mpratt** [comment]
Version 2.0.25 now has support for locale parameter on the facebook provider!
Vielen Dank @skyfox69 !
